test/test_solvers.jl

```diff
@@ -8,6 +8,8 @@ function tests()
         ("lbfgs", lbfgs),
         ("tron", tron),
         ("R2", R2),
+        ("fomo_r2", fomo),
+        ("fomo_tr", (nlp; kwargs...) -> fomo(nlp, step_backend = JSOSolvers.tr_step(); kwargs...)),
```

Comment:
```suggestion
        ("fomo_tr", (nlp; kwargs...) -> TR(nlp; kwargs...)),
```
?

Reply:
Oops, I mixed up things when rebasing.

Reply:
Shouldn't `fomo` be tested for both trust region and r2 step ?
Note that `fomo` uses momentum but not `TR`

Reply:
Yes, it should. I guessed from a super quick reading that `TR` was `fomo(nlp, step_backend = JSOSolvers.tr_step()`.